pkg/function/create_volume_from_snapshot_test.go

```diff
@@ -19,6 +19,7 @@ import (
 	"encoding/json"
 	"fmt"
 
+	"gopkg.in/check.v1"
```

Comment:
```suggestion
```
Already imported in the next line

Reply:
ohh.
go_lint passed somehow. I will fix this.